### PR TITLE
Hotreload of player-cli & Documentation

### PR DESCRIPTION
**Pull request by ChSotiriou**
- Added ATAKA_HOST in ctfconfig
- When hot reloading ataka (./ataka-cli reload) the ataka-player-cli.pyz gets repackeged.
- The players can reload their local instance by running the player-cli (atk reload)
- Documentation

### CODE REVIEW

**Comment by ChSotiriou** [comment]
Added bundling player-cli dependencies inside the Python Zipped Executable

File: README.md [hunk 1]
```diff
@@ -2,4 +2,228 @@
 
 Runs exploits, fast.
 
-See `.env` variable for ctf selection. See ataka/ctfconfig/ for ctf config code. This directory is mounted into the docker container, not copied. Edits can be applied via hot-reload.
+> See `.env` variable for ctf selection. See ataka/ctfconfig/ for ctf config code. This directory is mounted into the docker container, not copied. Edits can be applied via hot-reload.
+
+# Server
+
+1. Edit `.env` file to set:
+    - **DATA_STORE**: Path to a folder to store player exploit related files.
```
Comment (DaClemens) [inline README.md:10]:
```suggestion
    - **DATA_STORE**: **Absolute** path to a folder to store player exploit related files.
```

Comment (DaClemens) [inline README.md]:
```suggestion
    - **DATA_STORE**: **Absolute** path to a folder to store player exploit related files.
```

Docker requires this

File: README.md [hunk 2]
```diff
@@ -2,4 +2,228 @@
...
+    - **USERID**: The user that has access to the data directory and has access to the docker socket.
```
Comment (DaClemens) [inline README.md:11]:
```suggestion
    - **USERID**: The `user:group` id tuple to use for ataka. Note that write access write access to both the docker socket and the data directory has to be provided. You'll want to set this to the user id of the owner of the data directory and the group id of the `docker` group.
```

File: README.md [hunk 3]
```diff
@@ -2,4 +2,228 @@
...
+    - **CTF**: The name of the ctfconfig to use.
+2. Edit the ctfconfig in `ataka/ctfconfig/`
+3. Run `docker-compose up -d --build`
+
+> When editing the config while ataka is running, run `./ataka-cli reload` to hot-reload the ctfconfig.
```
Comment (DaClemens) [inline README.md:16]:
```suggestion
> The ctfconfig is mounted into the containers. When editing the config while ataka is running, run `./ataka-cli reload` to hot-reload the ctfconfig.
```

**Review by DaClemens — APPROVED** [review]

**Comment by DaClemens** [comment]
Merged manually

**Pull request closed without merging**